addons/account/tests/test_account_move_line_tax_details.py

```diff
@@ -946,7 +947,7 @@ def test_multiple_same_tax_lines_multi_currencies_manual_edition(self):
         })
         tax_rep = percent_tax.refund_repartition_line_ids.filtered(lambda x: x.repartition_type == 'tax')
 
-        move = self.env['account.move'].create({
+        move = self.env['account.move'].with_context(skip_invoice_sync=True).create({
```

Comment:
Because this is not a move that can be created if you put the `tax_ids` on the lines, without changing the tax amounts manually afterwards. By simplicity, I just don't sync.